main.go

```diff
@@ -66,20 +66,22 @@ var commands = []*discordgo.ApplicationCommand{
 	},
 }
 
-const TOKEN_DISCORD = "0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
-
 var (
 	App   = flag.String("app", "", "Application ID")
 	Guild = flag.String("guild", "", "Guild ID")
+	Token = flag.String("token", "", "Discord Bot Token")
 )
 
 func main() {
 	flag.Parse()
 	if *App == "" {
 		log.Fatal("application id is not set")
 	}
+	if *Token == "" {
+		log.Fatal("discord bot token is not set")
+	}
```

Comment:
Finding at: https://github.com/ledo01-test-org/discord-bot/blob/f40f4a7b69eb640a4c25b1f5bb392f4ba2bfec81/main.go#L81


```suggestion
func blob() {
	flag.Parse()
	if *App == "" {
		log.Fatal("application id is not set")
	}
	if *Token == "" {
		log.Fatal("discord bot token is not set")
	}
```

```diff
+ 	_, err := session.ApplicationCommandBulkOverwrite(*App, *Guild, commands)
-	if err != nil {
 		log.Fatalf("could not register commands: %s", err)
 	}
```